files/rsyslog/55-elasticsearch.conf

```diff
@@ -62,9 +62,10 @@ ruleset(name="try_es") {
             queue.maxFileSize=`echo $ES_QUEUE_MAXFILESIZE`
             queue.checkpointInterval=`echo $ES_QUEUE_CHECKPOINTINTERVAL`
             queue.syncqueuefiles="on"
+            queue.size=`echo $ES_QUEUE_SIZE`
             action.resumeRetryCount="-1"
             )
-            # not yet supported - action.resumeIntervalMax=`echo $ES_RETRY_WAIT`
+            # not yet supported - RHEL 8.2 rebase - action.resumeIntervalMax=`echo $ES_RETRY_WAIT`
```

Comment:
Out of curiosity... OAL vendored_src does not contain the latest rsyslog source codes?

Reply:
it contains the latest rsyslog source code from rhel8

Reply:
Then, we could uncomment???

Reply:
rhel 8.2 has not been released yet . . .